packages/round-manager/src/features/round/RoundDetailForm.tsx

```diff
@@ -234,326 +284,411 @@ export function RoundDetailForm(props: RoundDetailFormProps) {
+                            }}
+                            inputProps={{
+                              id: "roundStartTime",
+                              placeholder: "",
+                              className:
+                                "block w-full border-0 p-0 text-gray-900 placeholder-grey-400 focus:ring-0 text-sm",
+                            }}
+                            isValidDate={disableBeforeApplicationStartDate}
+                            utc={true}
+                            dateFormat={"YYYY-MM-DD"}
+                            timeFormat={"HH:mm UTC"}
+                          />
+                        )}
+                      />
+                      <div className="absolute inset-y-0 right-0 pr-3 flex items-center pointer-events-none">
+                        <svg
+                          xmlns="http://www.w3.org/2000/svg"
+                          className="h-5 w-5"
+                          viewBox="0 0 20 20"
+                          fill="currentColor"
+                        >
+                          <path
+                            fillRule="evenodd"
+                            d="M6 2a1 1 0 00-1 1v1H4a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V6a2 2 0 00-2-2h-1V3a1 1 0 10-2 0v1H7V3a1 1 0 00-1-1zm0 5a1 1 0 000 2h8a1 1 0 100-2H6z"
+                            clipRule="evenodd"
+                          />
+                        </svg>
+                      </div>
                     </div>
+                    {errors.roundStartTime && (
+                      <p
+                        className="text-xs text-pink-500"
+                        data-testid="round-start-date-error"
+                      >
+                        {errors.roundStartTime?.message}
+                      </p>
+                    )}
+
+                    {/* Round end date */}
+                    {roundCategory == RoundCategory.Direct && (
+                      <Controller
+                        control={control}
+                        name="roundEndTimeDisabled"
+                        render={({ field }) => (
+                          <div className="flex items-center mt-2">
+                            <input
+                              id="noEndDate"
+                              type="checkbox"
+                              className="h-4 w-4 text-indigo-600 focus:ring-indigo-500 border-gray-300 rounded"
+                              checked={field.value}
+                              onChange={(e) => {
+                                field.onChange(e.target.checked);
+                                if (e.target.checked) {
+                                  // reset input and trick type validation
+                                  // for direct round, when date is empty it will be set to maxDate
+                                  setValue(
+                                    "roundEndTime",
+                                    null as unknown as Date
```

Comment:
Was it not possible to have the `roundEndTime` value as `null | Date`?

Reply:
it was, but if I changed it to be null | Date in the Round type, there were a lot of places breaking because of it.
I could have changed/fixed them, but I didn't for the following reasons:

* This trick is not the best, I admit it, but it is well contained as we are handling all the dates before we send the creating the round tx. This `null` should not have impact beyond the creating form. 
* Setting the roundEndTime as null, would require a LOT of changes in many parts of the code as the Round type is widely used. And to be honest I didn't want to add more noise to this PR as it is a bit complex and already big enough.
* Not sure if making roundEndTime `null | Date` is something that we want for the rest of the app. This is the only place were it need to be possibly `null`. Perhaps an alternative could be defining a new Round type and overriding the roundEndTime.